src/components/playerOverview/PlayerDetails.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { useLoaderData } from "react-router-dom";
+import { Loading } from "../Loading";
+import { getMatchesByPlayerName } from "../../redux/statsSelectors";
+import { useSelector } from "react-redux";
+import { AppState } from "../../redux/rootReducer";
+import { Flex, Heading, Text } from "@chakra-ui/react";
```

Comment:
nit, this should go after all of the "react-..." dependencies. and then an empty line before any imports that come from the local project. 

If you could, please group together all "react-..." dependencies together.